src/api/txMining.js

```diff
@@ -33,8 +33,8 @@ const txMiningApi = {
     }
     return txMiningRequestClient(resolve).post('submit-job', postData).then((res) => {
       resolve(res.data)
-    }, (res) => {
-      return Promise.reject(res);
+    }, (error) => {
```

Comment:
I noticed this object is not a response, it's actually an error object that contains the response (among other things)